packages/ckeditor5-typing/src/texttransformation.js

```diff
@@ -19,11 +19,11 @@ const TRANSFORMATIONS = {
 	trademark: { from: '(tm)', to: '™' },
 
 	// Mathematical:
-	oneHalf: { from: '1/2', to: '½' },
-	oneThird: { from: '1/3', to: '⅓' },
-	twoThirds: { from: '2/3', to: '⅔' },
-	oneForth: { from: '1/4', to: '¼' },
-	threeQuarters: { from: '3/4', to: '¾' },
+	oneHalf: { from: /(^|[^/a-z0-9])(1\/2)([^/a-z0-9])/i, to: [ null, '½', null ] },
+	oneThird: { from: /(^|[^/a-z0-9])(1\/3)([^/a-z0-9])/i, to: [ null, '⅓', null ] },
+	twoThirds: { from: /(^|[^/a-z0-9])(2\/3)([^/a-z0-9])/i, to: [ null, '⅔', null ] },
+	oneForth: { from: /(^|[^/a-z0-9])(1\/4)([^/a-z0-9])/i, to: [ null, '¼', null ] },
+	threeQuarters: { from: /(^|[^/a-z0-9])(3\/4)([^/a-z0-9])/i, to: [ null, '¾', null ] },
```

Comment:
```suggestion
	oneHalf: { from: /(^|[^/a-z0-9])(1\/2)([^/a-z0-9])$/i, to: [ null, '½', null ] },
	oneThird: { from: /(^|[^/a-z0-9])(1\/3)([^/a-z0-9])$/i, to: [ null, '⅓', null ] },
	twoThirds: { from: /(^|[^/a-z0-9])(2\/3)([^/a-z0-9])$/i, to: [ null, '⅔', null ] },
	oneForth: { from: /(^|[^/a-z0-9])(1\/4)([^/a-z0-9])$/i, to: [ null, '¼', null ] },
	threeQuarters: { from: /(^|[^/a-z0-9])(3\/4)([^/a-z0-9])$/i, to: [ null, '¾', null ] },
```

This `$` is needed there to act only on the closest character typed and not some remote one:

![](https://user-images.githubusercontent.com/1232187/133578029-6ab3d892-28d5-4578-9e85-e552db2a3d00.gif)

Reply:
And this should have some tests.